requirements-dev.txt

```diff
@@ -14,3 +14,4 @@ splinter>=0.13.0
 -e git+https://github.com/ckan/ckantoolkit@release-0.0.4#egg=ckantoolkit
 -e git+https://github.com/ckan/ckanapi@ckanapi-4.3#egg=ckanapi
 -e git+https://github.com/ckan/ckanext-scheming@release-1.2.0#egg=ckanext-scheming
+-e git+https://github.com/ckan/ckanext-harvest.git@v1.2.0#egg=ckanext-harvest
```

Comment:
would be awesome if this file was built dynamicly off  the vars opendata variables.